codelabs/codelab-03/lat_test.sh

```diff
@@ -0,0 +1,11 @@
+curl -s -o /dev/null -w \
```

Comment:
How about this:

```
ERR_MSG='Provide a URL as a parameter: ./lat_test.sh https://example.com/an/image.jpeg'
URL=${1:?$ERR_MSG}

echo "Testing: $URL"

curl -s -o /dev/null -w \
'
time_namelookup: %{time_namelookup} 
time_connect:  %{time_connect}
time_appconnect:  %{time_appconnect}
time_pretransfer:  %{time_pretransfer}
time_redirect:  %{time_redirect}
time_starttransfer:  %{time_starttransfer}
-
time_total: %{time_total}\n' \
$URL
```

That way, folks don't need to submit an updated lat_test.sh (it'll just print the URL).

We should also require that folks screenshot this command running on a few other images, to test various latencies.